packages/java/maven-plugin/pom.xml

```
@@ -0,0 +1,88 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>com.vaadin.fusion</groupId>
+    <artifactId>fusion-project</artifactId>
+    <version>1.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>generator-maven-plugin</artifactId>
```

```suggestion
  <artifactId>fusion-maven-plugin</artifactId>
```

Turns out, this plugin package name prefix can be used as a plugin prefix when running the goals. Therefore with `fusion-maven-plugin` users can run `mvn fusion:generate` in their project, whereas right now it’s `mvn generator:generate`, which is a bit too generic, I think.